template/src/pages/Login/index.tsx

```diff
@@ -1,16 +1,146 @@
-      </button>
-    </div>
+    <LoginWrapper>
+      <Flex
+        data-testid="login"
+        direction="column"
+        justifyContent="center"
+        w={['auto', 'md']}
+        boxShadow="xl"
+        p={['7', '10']}
+      >
+        <Center>
+          <Text fontSize="xl" fontWeight="semibold">
+            Fill the fields below to login
+          </Text>
+        </Center>
+        <form data-testid="login-form" onSubmit={handleSubmit(submitAction)}>
+          <Box mt="10">
+            <ControlledInput
+              aria-label="email"
+              focusBorderColor="green.600"
+              errorBorderColor="red.500"
+              label="Email address"
+              errorMessage={errors?.email?.message}
+              register={register('email', {
+                required: 'Email must be provided.',
+                pattern: {
+                  value: EMAIL_REGEX,
+                  message: 'Insert a valid email address.',
+                },
+              })}
+            />
+          </Box>
+          <Box mt="10">
+            <ControlledInput
+              type={isPasswordVisible ? 'text' : 'password'}
+              aria-label="password"
+              label="Password"
+              errorMessage={errors?.password?.message}
+              focusBorderColor="green.600"
+              errorBorderColor="red.500"
+              register={register('password', {
+                required: 'Password must be provided.',
+                minLength: {
+                  value: 7,
+                  message: 'The password must have at leat 7 characters.',
+                },
+              })}
+              icon={
+                <IconButton
+                  variant="outline"
+                  onClick={handlePasswordVisibility}
+                  colorScheme="gray"
+                  borderRadius="50%"
+                  border="none"
+                  aria-label="Visibilidade"
+                  fontSize="20px"
+                  icon={!isPasswordVisible ? <ViewIcon /> : <ViewOffIcon />}
+                />
+              }
+            />
+          </Box>
+          <Flex mt="2" mb="10" justifyContent="space-between">
+            <Checkbox colorScheme="green">Remember me</Checkbox>
+            <Link href="" color="green.500">
+              Forgot my password
+            </Link>
+          </Flex>
+          <Button
+            data-testid="submit-login"
+            disabled={!isValid}
+            isLoading={isSubmitting}
+            type="submit"
+            w="100%"
+            mt="0"
+            backgroundColor="green.500"
+            color="white"
+            _hover={{ backgroundColor: '#276749' }}
+          >
+            Login
+          </Button>
+          {errorSubmitMessage.length > 0 && (
+            <Box mt="3" textAlign="center">
+              <Text color="red.500" fontWeight="semibold">
+                {errorSubmitMessage}
+              </Text>
+            </Box>
+          )}
```

Comment:
```suggestion
          {errorSubmitMessage.length > 0 ? (
            <Box mt="3" textAlign="center">
              <Text color="red.500" fontWeight="semibold">
                {errorSubmitMessage}
              </Text>
            </Box>
          ) : null}
```
https://github.com/t10d/cra-template-kingdom/pull/20#discussion_r614090996